src/client/src/constants/tetriminos.ts

```diff
@@ -0,0 +1,53 @@
+import _ from 'lodash';
+
+export const blockShape = {
+  I: [
+    [1, 1, 1, 1],
+  ],
+  L: [
+    [0, 0, 1],
+    [1, 1, 1],
+  ],
+  J: [
+    [1, 0, 0],
+    [1, 1, 1],
+  ],
+  Z: [
+    [1, 1, 0],
+    [0, 1, 1],
+  ],
+  S: [
+    [0, 1, 1],
+    [1, 1, 0],
+  ],
+  O: [
+    [1, 1],
+    [1, 1],
+  ],
+  T: [
+    [0, 1, 0],
+    [1, 1, 1],
+  ],
+};
```

Comment:
L == J
Z == S

Right? They are just rotations of each other

![image](https://user-images.githubusercontent.com/33632673/94268741-eba88800-ff3d-11ea-9179-454aa2cb04a5.png)